rudder-core/src/main/scala/com/normation/rudder/repository/EventLogRepository.scala

```diff
@@ -342,46 +340,18 @@ trait EventLogRepository {
     )
   }
 
-
   /**
    * Node properties: heartbeat, agent run, properties
    */
-  def saveModifyNodeAgentRun(
-      modId: ModificationId
-    , principal: EventActor
-    , modifyDiff: ModifyNodeAgentRunDiff
-    , reason:Option[String]) = {
-    saveEventLog(
-        modId
-      , eventLogFactory.getModifyNodeAgentRunFromDiff (
-            principal  = principal
-          , modifyDiff = modifyDiff
-          , reason = reason
-        )
-    )
-  }
-  def saveModifyNodeHeartbeat(
-      modId: ModificationId
-    , principal: EventActor
-    , modifyDiff: ModifyNodeHeartbeatDiff
-    , reason:Option[String]) = {
-    saveEventLog(
-        modId
-      , eventLogFactory.getModifyNodeHeartbeatFromDiff (
-            principal  = principal
-          , modifyDiff = modifyDiff
-          , reason = reason
-        )
-    )
-  }
-  def saveModifyNodeProperties(
-      modId: ModificationId
-    , principal: EventActor
-    , modifyDiff: ModifyNodePropertiesDiff
-    , reason:Option[String]) = {
+  def saveModifyNode(
```

Comment:
i think i prefered the name NodeProperties, as modify node leads me to think we are changing the node, and not properties attached to it